### DISCUSSION TITLE
My favorite Tech + Entertainment Videos 📺 ⏯️ 🎉 😲

### DISCUSSION THREAD

**Discussion by josencf** [deepgram/community#83] [General help]
Hi!

So I'm venturing two guesses here:
(1) Most of us on this forum love tech at least a little
(2) Most of us on this forum also enjoy fun, laughs, and getting our jaws dropped

If both of these guesses are right for you, **here's a list of pure-entertainment videos where people use technology for nothing but chaos and fun**. 

[Michael Reeves - I Gave My Goldfish $50,000 to Trade Stocks](https://www.youtube.com/watch?v=USKD3vPD6ZA)
[Michael Reeves - I Let a Twitter Bot decorate my Room](https://www.youtube.com/watch?v=hBP-NzOadL0)
[Slow Mo Guys - Diving into 1000 Mouse Traps in Slow Motion](https://www.youtube.com/watch?v=ewGAmiLuYCw)
[Tom Scott - We Sent Garlic Bread to the Edge of Space, then ate it](https://www.youtube.com/watch?v=c8W-auqg024)
[Defunctland - Disney's Fastpass: A Complicated History](https://www.youtube.com/watch?v=9yjZpBq1XBE)

Btw, if anyone wants to share their favorite chaotic tech videos too, share in the thread! I'm always looking for something to watch while I eat my midnight snacks 🥳

**Comment by michaeljolley** [comment]
Hilarious. I'm still in awe of Colin Furze building a [tunnel and bunker in his back yard](https://www.youtube.com/watch?v=8RVLim5tVwU).

**Comment by BekahHW** [comment]
I love these. My kids 13-6 are always sharing amazing random facts and videos with me and I love it. Part of the reason is I haven't considered them. The other part is the imagination that happens here, thinking about things you wouldn't normally.
